modules/ROOT/pages/insight.adoc

```diff
@@ -180,7 +180,12 @@ image::chexcmsg.png[CHExcMsg]
 
 After investigating the logs and resolving the issue that caused the transaction failure – for example, by purchasing more API capacity  – you can simply replay the transaction.
 
-*Note*: Replay only works for flows that have inbound endpoints.
+[NOTE]
+====
+At the moment, the replay feature is only available for Mule 3 applications.
```

Comment:
Change wording to:

Currently, the replay feature is supported only for Mule 3 applications and for flows that have inbound endpoints.